pgadmin_macros/function_get_row_count.sql

```diff
@@ -0,0 +1,17 @@
+--for testing: 
+--rescu.volumes_15min
+
+DROP FUNCTION get_row_count(text);
+CREATE OR REPLACE FUNCTION get_row_count(IN schema_table text)
+RETURNS TEXT AS
+$$
+DECLARE
+  row_count TEXT;
```

Comment:
could it be an int?

Reply:
I changed it to a comma formatted text cause I'm always bothered counting the digits